### DISCUSSION TITLE
Welcome to dream Discussions!

### DISCUSSION THREAD

**Discussion by Benefitspace** [Benefitspace/dream#1] [General]
现在的问题是我的deram应该是被锁死了，我不明白错误发生在哪个位置，有人可以给与告知么？

**Comment by dupz49** [comment]
有电报那些的社交群吗？

**Comment by wonder-all** [comment]
能不能搞定？   我是hightbiger   搞不定的话给我们社区点份额，我帮你想办法看下能不能修复，我觉得你项目还可以。就是技术还差点

**Reply by Benefitspace** [reply]
合约出问题了，LP被卡住无法解冻，我尝试绕过pancakeswap直接在合约上解除流动性，但是失败了；不知道可否解决？   给份额没问题的，本来就是要拿部分出来做推广，我是真的有个梦想，代码层面我也是去参考成功案例的，这次出问题应该是V池的切换造成的，我把路由器代码写死在合约里面，写的还是V1的代码，由于合约的不可篡改特性，我没办法进行相应的改动造成现在这样的情况。因为解除LP要调用合约，所以变成一个锁死的死循环；  如果能解决最好，解决不了我也会想办法重启项目，不会太监的，希望大家放心，我本人匿名做项目只是不想惹麻烦，可是我并不是为了骗钱才做的项目。

**Reply by wonder-all** [reply]
我看你已经解决了，个人觉得既然合约没法改动了，最好是也去一池添加下流动性，在这之前liquidityfee=0吧，大家交易可以少烧5%，算是福利，然后  0x015078950f7B7D7c8060742Cd13b565A9929FEe8   发点代币过来吧，我拿去送我社区的小伙伴，没问题吧？

**Reply by Benefitspace** [reply]
感谢，我发个公告些，空投应该的。

**Reply by westlife29BTC** [reply]
我也让我的社区参与了，希望也能得到空投。
0xc29C546064E8fB5506C5e5Beaf1594866Df7177e

**Reply by wonder-all** [reply]
成，等你好消息，建议是做个推特或者电报群，毕竟用GitHub的都是程序员

**Reply by westlife29BTC** [reply]
说得好像你是程序员一样，哈哈。

**Reply by musk121** [reply]
> 感谢，我发个公告些，空投应该的。
终于解决了 前天往群里给小伙伴推荐了梦想，结果发现买入不了 出问题了 都说梦想跑路了被嘲笑了一番，希望能得到空投0xe1bbcee8510198279e40dbf36ea954d79bbe9e8d

**Reply by wonder-all** [reply]
> 
> 
> 说得好像你是程序员一样，哈哈。
额，，，我以前学过div+css的男人好吧

**Reply by Benefitspace** [reply]
我打算去一池增加下流动性，这样后续价格才会稳步上去，现在这样流动性池子增加速度太慢，价格波动太大

**Reply by Benefitspace** [reply]
> 我看你已经解决了，个人觉得既然合约没法改动了，最好是也去一池添加下流动性，在这之前liquidityfee=0吧，大家交易可以少烧5%，算是福利，然后 0x015078950f7B7D7c8060742Cd13b565A9929FEe8 发点代币过来吧，我拿去送我社区的小伙伴，没问题吧？

空投等我加完池子给你，现在给如果有人砸盘价格会跌的比较凶，见谅

**Comment by sean-go-ai** [comment]
> 现在的问题是我的deram应该是被锁死了，我不明白错误发生在哪个位置，有人可以给与告知么？


已建立群組了 我們現在去瘋狂宣傳去 https://t.me/dream_BSC

https://discord.gg/s4TzMfMRsf

**Comment by westlife29BTC** [comment]
希望给社区一部分空投，可以发到我钱包，我来在discord里面建立一个空投频道，分发给早期支持者
0xc29C546064E8fB5506C5e5Beaf1594866Df7177e

**Comment by westlife29BTC** [comment]
另外2个微信群已经超过300人加入

**Comment by wonder-all** [comment]
https://www.youtube.com/watch?v=1cx0DNDqWg8    0xe8dbBa8ac8411Ee9F51FB4A2FEBA1A10243dba84  xuanli 的，昨天他做的推广视频
